tsconfig.json

```diff
@@ -12,7 +13,12 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "jsx": "preserve"
+    "jsx": "preserve",
+    "paths": {
+      "@/core/*": ["core/*"],
+      "@/layout/*": ["components/layout/*"],
+      "@/markdown/*": ["components/markdown/*"],
```

Comment:
These last 2 may be a bit too specific and become out of date every time we add a component to the website.

I would suggest changing it to @/components instead.

Reply:
Aight, I'll do that soon :D